pkg/services/sqlstore/stats_test.go

```diff
@@ -18,41 +18,51 @@ func TestStatsDataAccess(t *testing.T) {
 
 			query := models.GetSystemStatsQuery{}
 			err := GetSystemStats(&query)
-			assert.Nil(t, err)
-			assert.Equal(t, query.Result.Users, int64(3))
-			assert.Equal(t, query.Result.Editors, 1)
-			assert.Equal(t, query.Result.Viewers, 1)
-			assert.Equal(t, query.Result.Admins, 3)
+			assert.NoError(t, err)
```

Comment:
Consider using `require.NoError(...)` which will end the test if it fails, otherwise it will continue and that could result in additional errors/panics.

Reply:
Done!

Reply:
Nice